### DISCUSSION TITLE
Push from action doesn’t trigger workflows

### DISCUSSION THREAD

**Discussion by asmitty92** [stefanzweifel/git-auto-commit-action#336] [Help]
I’m using this action to push changes from my linter/formatter. Once that push is complete, I have an on: push workflow that should run my unit tests. However, the push done by this action doesn’t trigger my unit test workflow. Is there a config setting I can set to ensure my unit test workflow runs?

**Comment by stefanzweifel** [comment]
@pythonclimber You have to create and configure a personal access token in order to do this. (This is a limitation of the default `GITHUB_TOKEN`)

You can read more about this here:
https://github.com/stefanzweifel/git-auto-commit-action?tab=readme-ov-file#commits-made-by-this-action-do-not-trigger-new-workflow-runs
